coderd/gitauth/config.go

```diff
@@ -315,19 +319,19 @@ func ConvertConfig(entries []codersdk.GitAuthConfig, accessURL *url.URL) ([]*Con
 
 		var oauthConfig OAuth2Config = oc
 		// Azure DevOps uses JWT token authentication!
-		if typ == codersdk.GitProviderAzureDevops {
+		if typ == codersdk.ExternalAuthProviderAzureDevops {
 			oauthConfig = &jwtConfig{oc}
 		}
 
 		cfg := &Config{
-			OAuth2Config:        oauthConfig,
-			ID:                  entry.ID,
-			Regex:               regex,
-			Type:                typ,
-			NoRefresh:           entry.NoRefresh,
-			ValidateURL:         entry.ValidateURL,
-			AppInstallationsURL: entry.AppInstallationsURL,
-			AppInstallURL:       entry.AppInstallURL,
+			OAuth2Config:           oauthConfig,
+			ID:                     entry.ID,
+			GitCloneRegex:          regex,
```

Comment:
Should this be domain/auth url regex instead? Git and clone seems very specific.

Reply:
Going to revert these renames for now. It'll make more sense in my future commit!